src/ProjectTemplates/test/BlazorWasmTemplateTest.cs

```diff
@@ -74,13 +74,14 @@ private void ValidatePublishedServiceWorker(Project project)
         }
 
         [ConditionalFact]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/34554", Queues = "Windows.10.Arm64v8.Open")]
         // LocalDB doesn't work on non Windows platforms
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
         public Task BlazorWasmHostedTemplate_IndividualAuth_Works_WithLocalDB()
             => BlazorWasmHostedTemplate_IndividualAuth_Works(true);
 
         [Fact]
```

Comment:
I think you need ConditionalFact for this to take effect, this has bitten me before.

Reply:
```suggestion
        [ConditionalFact]
```